vespalib/src/vespa/vespalib/net/tls/authorization_mode.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "authorization_mode.h"
+
+#include <iostream>
+
+namespace vespalib::net::tls {
+
+const char* to_string(AuthorizationMode mode) noexcept {
```

Comment:
to_string combined with string values that does not match the 'from string' behavior can be somewhat confusing. Consider using symmetric string mapping or renaming this function.

Reply:
Renamed to `enum_name`, PTAL